src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -632,9 +654,12 @@ const MetaTraderUI = (() => {
             $view_2_button_container.setVisibility(1);
         } else if (step === 3) {
             $form.find('input').not(':input[type=radio]').val('');
-
-            const $view_3_button_container = $form.find('#view_3-buttons');
-
+            let $view_3_button_container;
+            if (should_set_trading_password) {
+                $view_3_button_container = $form.find('#view_3-buttons_new_user');
+            } else {
+                $view_3_button_container = $form.find('#view_3-buttons_existing_user');
+            }
```

Comment:
```suggestion
            const $view_3_button_container= should_set_trading_password ? $form.find('#view_3-buttons_new_user') : $form.find('#view_3-buttons_existing_user');
```

Reply:
I think the if statement is more readable in this case.